src/azure-cli/azure/cli/command_modules/databoxedge/generated/_client_factory.py

```diff
@@ -0,0 +1,44 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+
+def cf_databoxedge_cl(cli_ctx, *_):
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    from azure.mgmt.databoxedge import DataBoxEdgeManagementClient
+    return get_mgmt_service_client(cli_ctx,
```

Comment:
use resource type not client for multiapi supported client factory

Reply:
modified